app/views/users/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="col-md-8 col-md-offset-2">
   <li><%= link_to "Log out", destroy_user_session_path, method: :delete %></li>
-
+  <li><%= link_to 'navigate to the admin panel', admin_panel_path if current_user.admin %></li>
```

Comment:
In the current solution, you are showing `<li></li>` if user is not an admin - you probably do not want to have that empty li there 😄